Document/0x05b-Basic-Security_Testing.md

```diff
@@ -1244,13 +1244,43 @@ For enterprise tools, see the section "Static Source Code Analysis" in the chapt
 
 #### Dynamic Analysis
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1240>
+Dynamic Analysis tests the release version of an application, in a similar fashion to black box application testing. The way the code works should be inferred and looked at from a higher level, unlike static analysis, which takes the application's code and tries to analyze how the application will behave. This allows the tester to properly identify:
+
+- Business logic flaws which extend over multiple services
+- Vulnerabilities in live environments
```

Comment:
Just say "Vulnerabilities in the tested environment". Testing could be executed in various environments